compressor/compressor.cpp

```diff
@@ -0,0 +1,177 @@
+#include <string>
+#include <fstream>
+#include <deque>
+#include <iostream>
+#include "Bit_cashier.cpp"
+//Solve the issue with weird representation of first bytes in a text file
+//
+const int window_width = 510;//Won't work if more
+const int byte_length = 8;
+const int encoded_length = 17;
+const int untouched_length = 9;
+using Byte = char;
```

Comment:
again better do not use char as Byte.
And why you use Byte alias in some cases and direct char in others?